File: net-examples/Aspose.Font.Examples/IExamples.cs
namespace Aspose.Font.Examples
{
    interface IExamples
    {
        string DataDir { get; }
        string OutputDir { get; }

        string LicenseDir { get; }

        string DataPath(string path);
        string DataPath(params string[] paths);
        string OutputPath(string path);

        void Run();
    }
}

File: java-examples/src/main/java/com/aspose/font/examples/IExamples.java
package com.aspose.font.examples;

public interface IExamples {
	
	String getDataDir();
	String getOutputDir();

	String getLicenseDir();

	String setDataPath(String path);
	String setDataPath(String ... path);	
	String setOutputPath(String path);

	void run() throws Exception;
}